content/en/blog/2023/announcing-notation-rc3.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Notation v1.0.0-RC.3 is available!
+author:  "Notation Release Team"
+date: 2023-03-08
+draft: false
+---
+
+The Notation maintainers are pleased to announce the release of Notation CLI v1.0.0-RC.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release introduces a breaking change to Notation. The default type of signature manifest is changed to image manifest. The flag `--signature-manifest` for `notation sign` command is experimental for users to store signatures using artifact manifest.
+
+Signing experience is continuously improved in this release by introducing new flags for signing artifacts using on-demand keys.
```

Comment:
Can we list what those improvements are?

Reply:
I think we have only one improvement in RC.3 which has been mentioned [below](https://github.com/notaryproject/notaryproject.dev/pull/151/files#diff-c635c2bfa13e5aa791046117c8881b81b9a0346b029052d00011e5759e59ef85R34).